meta-resin-common/recipes-core/packagegroups/packagegroup-resin.bb

```diff
@@ -22,6 +22,5 @@ RDEPENDS_${PN} += " \
     resin-state-reset \
     resinhup \
     aufs-util \
-    ${@bb.utils.contains('RESIN_CONNECTABLE', '1', 'resin-connectable', '', d)} \
     ${@bb.utils.contains('RESIN_CONNECTABLE', '1', 'resin-provisioner', '', d)} \
```

Comment:
As @willnewton said above, I don't understand who will/how should RESIN_CONNECTABLE be define(d) so that the above line makes sense. I thought this Pr completely removes RESIN_CONNECTABLE from meta-resin.